src/modules/form/hooks/usePreloadPicklists.tsx

```diff
@@ -94,7 +90,10 @@ const usePreloadPicklists = ({
 
   return {
     fetch,
-    loading: loading && Object.keys(data).length === 0,
+    loading:
+      loading &&
+      pickListTypesToFetch.length > 0 && // always loading:false if there was no fetch
```

Comment:
I thought it would be nicer if the `loading` state defaulted to false, and only got set to true if a fetch started. However that didn't work as expected because of how this component is calling setState from a callback called by a useEffect... it's a different pattern from usual, I didn't want to touch it too much. This gives the same desired result, which is that forms with no pick lists never enter a `loading` state.